frontend/src/components/dashboard/visitors_flow/VisitorsFlowDiagram.vue

```diff
@@ -86,20 +88,31 @@
                 }
             },
 
+            createExits() {
+                this.exits = [];
+                let lastNodeId = this.nodes.slice(-1)[0].id;
+
+                for (let sourceInd = 5; sourceInd < lastNodeId; sourceInd++) {
+                    let sourceId = this.nodes[sourceInd].id;
+                    let value = Math.floor(Math.random() * 50);
+                    let exit = { source: sourceId, value: value, index: sourceId - 1 };
+                    this.exits.push(exit);
```

Comment:
All keys in local state are reactive, so each push will cause props evaluation. It's better to define variable, and put in local state at the end